src/test/rustdoc-ui/coverage/output-format-fail-check.rs

```diff
@@ -0,0 +1,4 @@
+// compile-flags:-Z unstable-options --output-format
```

Comment:
What was this supposed to test? Was there meant to be something passed to `--output-format`? As this test is in the `coverage` directory, was `--show-coverage` supposed to be passed here?